hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -28,10 +28,14 @@ HologramDataSourceAvailabilityFetcher::HologramDataSourceAvailabilityFetcher() {
         absl::GetFlag(FLAGS_chipper_batch_job_cell);
     system_to_cell_[System::CHIPPER_GDPR] = 
         absl::GetFlag(FLAGS_chipper_gdpr_batch_job_cell);
+    AcquireConfig(absl::GetFlag(FLAGS_hologram_source_config_file_path));
```

Comment:
ooc, as discussed, I thought this should be in the constructor?

Reply:
this is the constructor